scripts/ldmx-env.sh

```diff
@@ -226,9 +232,13 @@ elif hash singularity &> /dev/null; then
     local csv_list="/tmp/.X11-unix"
     for dir_to_mount in "${LDMX_CONTAINER_MOUNTS[@]}"; do
       csv_list="$dir_to_mount,$csv_list"
+    done
+	local env_list
```

Comment:
Just fixing the tabs
```suggestion
    local env_list
```